src/app.ts

```diff
@@ -16,9 +17,26 @@ app.use(cors());
 // Hello World endpoint
 app.get("/", (_req: Request, res: Response) => {
   res.json({
-    title: "Job Application Frontend",
+    title: "Job Application Backend API",
     message: "Welcome to the Job Application System",
+    version: "1.0.0",
+    endpoints: {
+      jobs: "/api/jobs",
+      openJobs: "/api/jobs/open",
+      jobById: "/api/jobs/:id",
+      jobsByCapability: "/api/jobs/capability/:capability",
+      jobsByBand: "/api/jobs/band/:band",
+      searchJobs: "/api/jobs/search",
+      closingSoon: "/api/jobs/closing-soon",
```

Comment:
good idea. I you want to keep it, it might be a good idea to add a instructions file for the agent to update this list whenever a new route is added. People can forget.